src/components/Settings/RouterPreferenceSettings/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { Trans } from '@lingui/macro'
+import Column from 'components/Column'
+import Radio from 'components/Radio'
+import { RowBetween, RowFixed } from 'components/Row'
+import Toggle from 'components/Toggle'
+import { RouterPreference } from 'state/routing/slice'
+import { useRouterPreference } from 'state/user/hooks'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+const Preference = styled(Radio)`
+  background-color: ${({ theme }) => theme.backgroundModule};
+  padding: 12px 16px;
+`
+
+const PreferencesContainer = styled(Column)`
+  gap: 1.5px;
+  border-radius: 12px;
+  overflow: hidden;
+`
+
+export default function RouterPreferenceSettings() {
+  const [routerPreference, setRouterPreference] = useRouterPreference()
+
+  const isAutoRoutingActive = routerPreference === RouterPreference.PRICE
```

Comment:
there should probs be a new `RouterPreference.AUTO` that is the default :3

Reply:
and for now it can just alias to be the same as `RouterPreference.API` :3

Reply:
`RouterPreference.PRICE` is just used internally for our token -> USDC trades to get a USD value. its kind of confusing that it's a "Router preference" type when it should really just be a "quote type"